File: Plus_Minus.py
def plusMinus(arr):
    p, n, z, x = 0, 0, 0, len(arr);
    for i in arr:
        if i > 0: p += 1
        elif i < 0: n += 1
        else: z += 1
    print("{0:.6f}\n{1:.6f}\n{2:.6f}".format(p / x, n / x, z / x))

File: Plus_Minus.cpp
void plusMinus(vector<int> arr) {
    double positive = 0, negative = 0, zero = 0;
    for(int i = 0; i < arr.size(); ++i) {
        if(arr[i] > 0) {
            positive++;
        } else if(arr[i] < 0) {
            negative++;
        } else {
            zero++;
        }
    }
    int x = arr.size();
    std::cout << setprecision(6) << positive / x << std::endl
     << negative / x << std::endl
     << zero / x;
}
